### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by alexandraBara**
- Removed flask
- fixed sphinx/markup
- Rearranged md files to have a better index file
- moved MIOpen docs under MIOpen tuning

Looking for place to publish the rendered html file, stay tuned

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,213 +1,249 @@
...
+  export TUNA_CELERY_JOB_BATCH_SIZE=<integer>
+  #rabbitMQ
+  export TUNA_CELERY_BROKER_HOST=localhost
+  export TUNA_CELERY_BROKER_USER=<username>
+  export TUNA_CELERY_BROKER_PWD=<pwd>
+  export TUNA_CELERY_BROKER_PORT=5672 #default
+  #redis
+  export TUNA_CELERY_BACKEND_HOST=localhost
+  export TUNA_CELERY_BACKEND_PORT=6379 #default
+  export gateway_ip=<gateway_ip>
```
Comment (cderb) [inline README.md:120]: can add a split here to indicate the gateway fields are for ipmi

**Review by cderb — APPROVED** [review]

File: tuna/miopen/doc/FinDocs.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+FIN Documentation
+==================
+
+How to run FIN inside a docker!
+-------------------------------
+
+To run Fin steps in tuna, a new docker needs to be created, that contains FIN, MIOpen
```
Comment (alexandraBara) [inline tuna/miopen/doc/FinDocs.md]:
```suggestion
To run MIFin steps in MITuna, a new docker needs to be created, that contains MIFin, MIOpen
```

File: tuna/miopen/doc/FinDocs.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+FIN Documentation
+==================
+
+How to run FIN inside a docker!
```
Comment (alexandraBara) [inline tuna/miopen/doc/FinDocs.md]:
```suggestion
How to run MIFin inside a docker!
```

File: tuna/miopen/doc/FinDocs.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+FIN Documentation
```
Comment (alexandraBara) [inline tuna/miopen/doc/FinDocs.md]:
```suggestion
MIFin Documentation
```

File: tuna/miopen/doc/Tuning.md [hunk 5]
```diff
@@ -129,14 +132,14 @@ the config_tags table. Jobs should have a compile and an eval fin step pair.
...
---fin_steps  - operations to be performed by fin (tuning handle into miopen)
--o           - only_applicable, will create a job for each applicable solver
--l           - reference text description
-```
+.. code-block::  
+
+  ./load_job.py --session_id 1 -t resnet50 --fin_steps miopen_perf_compile,miopen_perf_eval -o -l reason  
+  --session_id - tuning session id  
+  -t           - config tag 
+  --fin_steps  - operations to be performed by fin (tuning handle into miopen)
```
Comment (alexandraBara) [inline tuna/miopen/doc/Tuning.md]:
```suggestion
  --fin_steps  - operations to be performed by MIFin (tuning handle into MIOpen)  
```

**Review by reidkwja — APPROVED** [review]

**Pull request merged**